src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -48,6 +51,10 @@ public ConfigurationFeatureDefinitionProvider(IConfiguration configuration)
             {
```

Comment:
I'm a little confused by _getFeatureDefinitionFunc. I don't see us assigning it dynamically. It could just be a method of the class.